src/ApiService/ApiService/onefuzzlib/notifications/GithubIssues.cs

```diff
@@ -3,15 +3,15 @@
 namespace Microsoft.OneFuzz.Service;
 
 public interface IGithubIssues {
-    Async.Task GithubIssue(GithubIssuesTemplate config, Container container, IReport reportable, Guid notificationId);
+    Async.Task GithubIssue(GithubIssuesTemplate config, Container container, IReport reportable, Guid notificationId, Uri instanceUrl);
 }
 
 public class GithubIssues : NotificationsBase, IGithubIssues {
 
     public GithubIssues(ILogger<GithubIssues> logTracer, IOnefuzzContext context)
     : base(logTracer, context) { }
 
-    public async Async.Task GithubIssue(GithubIssuesTemplate config, Container container, IReport reportable, Guid notificationId) {
+    public async Async.Task GithubIssue(GithubIssuesTemplate config, Container container, IReport reportable, Guid notificationId, Uri instanceUrl) {
```

Comment:
You have access to `_context` in this function so we don't need to change the signature. You can get the isntanceUrl the same way you got it in notificationoperations by doing `_context.Creds.GetInstanceUrl()`

Reply:
Thanks for pointing this out :)